pageserver/src/tenant/timeline.rs

```diff
@@ -3457,14 +3457,14 @@ impl Timeline {
         // "gaps" in the sequence of level 0 files should only happen in case
         // of a crash, partial download from cloud storage, or something like
         // that, so it's not a big deal in practice.
-        level0_deltas.sort_by_key(|l| l.get_lsn_range().start);
+        level0_deltas.sort_by_key(|l| l.layer_desc().lsn_range.start);
```

Comment:
This should probably be [sort_by_cached_key](https://doc.rust-lang.org/stable/std/primitive.slice.html#method.sort_by_cached_key) because we do a virtual method call per each. But no matter, it's probably little compared to all of our clones. And we don't really have compaction benchies.

Reply:
And we only have 10 of these, so might not be hugely faster. But soon we will no longer have dynamic dispatch and the world will be perfect :)